integration/cli/build.noembed.test.ts

```diff
@@ -48,8 +48,7 @@ describe("build", () => {
           const argz = [`--no-embed`].join(" ");
           const build = await pepr.cli(testModule, { cmd: `pepr build ${argz}` });
           expect(build.exitcode).toBe(0);
-          expect(build.stderr.join("").trim()).toContain("Error: Cannot find module");
-          expect(build.stdout.join("").trim()).toContain("");
+          expect(build.stdout.join("").trim()).toContain("Module built successfully at");
```

Comment:
```suggestion
          expect(build.stderr.join("").trim()).toContain("");
          expect(build.stdout.join("").trim()).toContain("Module built successfully at");
```
Can we add a positive assert on stderr back in, just to assert that nothing is coming out?

If I hadn't had something like that when I was originally implementing this test then I wouldn't have caught the erroroneous output & putting an assert empty back in will be sure that we can catch any similar problems in the future.